jit/v2/JIT_v2.0.0.yaml

```
@@ -3279,13 +3281,11 @@ components:
             - `PLN` (Planned)
             - `REQ` (Requested)
 
-            **Condition:** `EST`, `PLN` and `ACT` are **only** to be used by the **provider** when sending a **Timestamp** to the **consumer**.
-
-            **Condition:** `REQ` is **only** to be used by the **consumer** when sending a **Timestamp** to the **provider**.
-
-            **Condition:** `ACT` can be used in a **Timestamp** for any **Port Call Service** except `portCallServiceType='MOVES'`
-
-            **Condition:** `EST`, `REQ`and `PLN` can be used in a **Timestamp** for any **Port Call Service** with `portCallServiceType` having one of the following values:
+            **Conditions:**
+            - `EST`, `PLN` and `ACT` are **only** to be used by the **provider** when sending a **Timestamp** to the **consumer**.
```

```suggestion
            - `EST`, `PLN` and `ACT` can **only** be used by the **provider** when sending a **Timestamp** to the **consumer**.
```
Preferably, we should consider to not use the constructs as 'are to be used', if more clear and easier options are available, when Shakespeare is not involved 😉 
Same for line 3286